usecase/usecase-webapi-xwalk-tests/samples/WebCL/res/COPYING

```diff
@@ -0,0 +1,5 @@
+index.html come from
+https://github.com/uebquite/webcl-demos
```

Comment:
Actually this is https://github.com/uebquite/webcl-demos/blob/master/Hello/index.html